src/store/actions/item-action.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { soapFetch } from '@zextras/carbonio-shell-ui';
+import { isNil, omitBy } from 'lodash';
+
+export const itemAction = createAsyncThunk(
```

Comment:
Does this function saves or interact with the store? I don't see any new reducer around